### ISSUE TITLE
Create new accessibility artifact

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#47915]

**Comment by shiragoodman** [comment]
@briandeconinck thanks for helping populate this ticket! I did make one small change to the tasks - I removed "Update documentation" as that is something that will happen in #47916

**Comment by artsymartha68** [comment]
This is so great. Commenting so I get updates :)

**Comment by briandeconinck** [comment]
@CherylEvans @sterkenburgsara This is ready for PR! A couple of notes:
- The [draft form for generating the accessibility testing artifact](https://github.com/briandeconinck/a11y-artifact-discovery/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-artifact.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5D) is available in a personal GitHub repo, since putting it in the va.gov-team repo would make it live for everyone.
- I'm going to have another a11y person do a test run too, but please give it a spin and let me know if anything doesn't make sense! Audience for this is mainly VFS teams that may not have an a11y person.
- The intro and instructions reference information on the ["Prepare for an accessibility staging review"](https://depo-platform-documentation.scrollhelp.site/developer-docs/prepare-for-an-accessibility-staging-review) page. That page is a little bit of a mess, and I'll want to clean it up as part of #47916. In particular, there are some WCAG 2.1 rules that got added to that page as though they should be part of foundational testing, but the content was never fully built out. For the purposes of this MVP, I'm choosing to leave those off the form and will be creating a backlog ticket to revisit. (They're important, but I'm not sure how useful it is to be asking teams to address them at this stage. Need to have some further conversation.)

Please let me know if you have any questions or suggested edits!

**Comment by shiragoodman** [comment]
hey @briandeconinck i did a quick overview (I will do a more detailed review once Cheryl completes hers) and I have only 1 comment at this time. Please add numbers to each of H2 section. For example, "1. Introduction", "2. Before you begin" etc. I think it'll just be easier to read and navigate, and to reference in the future. If you disagree, please let me know and we can discuss. I also think we want to test what the completed YAML format will look like since that is what you'll be reviewing.

**Comment by briandeconinck** [comment]
@shiragoodman Numbers added! I agree, that's going to make referencing sections easier with teams.

I have done a quick check of what a submitted issue will look like and I think it will be clear enough and useful enough to me (and to the team that may want to go back to reference it). But we'll check again when Cheryl and Sarah do their test run.

**Comment by CherylEvans** [comment]
@briandeconinck I would like to commend you on the great job you have done!  Your attention to detail, and the specificity with which you laid out your case was thoughtful and intentional.  :-)
I found only a few issues:
[Draft form for generating the accessibility testing artifact](https://github.com/briandeconinck/a11y-artifact-discovery/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-artifact.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5Durl):

1. In accordance with [design.va.gov guidance](https://design.va.gov/content-style-guide/punctuation#em-and-en-dashes-hyphensurl), we avoid using exclamation points in our content.  I assume this is going to live on the Platform website at some point, so it is probably a good idea to remove them.  I personally like EP's for emphasis, but for consistency, we should follow the existing guidance. lol.
2. Number 4., I recommend capitalizing "axe" because it is the beginning of a sentence.
3. I recommend capitalizing "axe" throughout, as it's a proper noun, much like "Cypress". 
4. I am not sure if "Review" needs to be capitalized throughout.  "Staging" is definitely a proper noun, and "review" is the modifier.  It is also a noun, but don't think it's part of the proper use.  I would **not** capitalize "Review" throughout.  I will check with SWC, and get their opinion.  
I will add more once I complete my review, but wanted to get this out there first.  Thanks Brian!

**Comment by shiragoodman** [comment]
hey @CherylEvans - thanks for your feedback . I think I have to disagree with your suggestion to capitalize axe. If you look at their [own site](https://www.deque.com/axe/browser-extensions/), they don't capitalize the name of their own product. We also have [existing pages on PW](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-testing-helper-functions) that reference axe and it's not capitalized there (however sometimes it's written aXe so I'm unsure of that).

**Comment by CherylEvans** [comment]
@briandeconinck Completed test #1
[Accessibility Testing for [47915, Cheryl E, TEST-1] #2](https://github.com/briandeconinck/a11y-artifact-discovery/issues/2url) - Basic test, no issues or contingencies.

**Comment by CherylEvans** [comment]
@briandeconinck I also wanted to add that what you presented in the [conversation thread accessibility ops slack channel](https://dsva.slack.com/archives/C023ZKDJJK1/p1664468503794349url) was a perfect start to the overall conversation and subsequent collaborative solution.

**Comment by CherylEvans** [comment]
@shiragoodman Thanks Shira!  I wasn't sure, as I see it written lower-case everywhere, but still it's a proper noun, but no biggie! :-)

**Comment by sterkenburgsara** [comment]
@briandeconinck I also submitted a [test ticket](https://github.com/briandeconinck/a11y-artifact-discovery/issues/3) (acting hat on like a real submitter). Also had no issues - the documentation here is very thorough and clear. I did some of these tests as a complete newbie and figured it out with guidance you provided. Looks really great!

My question regarding the template were: 
* should teams try and resolve any issues they find before review? Or discuss them with you at Staging?
* Is there a way to indicate whether something will be designated as launch blocking?

**Comment by briandeconinck** [comment]
@sterkenburgsara Good questions!
- Ideally yes, although in some instances they might need extra guidance on how to resolve. I'll add a note about that to the intro.
- That's harder to say, as something like keyboard nav testing can reveal a wide variety of issues -- some a big deal, some less so.
